client/src/components/Header.jsx

```diff
@@ -1,8 +1,17 @@
-
+import LogoMännchen from "../images/LogoMännchen3.svg"
```

Comment:
Using "ä" in javascript looks at least dangerous :o

Reply:
Thanks, you right... i will change this